src/Codeception/Lib/Driver/Db.php

```diff
@@ -288,4 +288,17 @@ protected function flushPrimaryColumnCache()
 
         return empty($this->primaryKeys);
     }
+    
+    public function update($table, array $data, array $criteria)
+    {
+
+        $set = [];
+        foreach ($data as $column => $value) {
+            $set[] = "{$column} = {$value}";
```

Comment:
I think that it would be better to quote column names witrh  `$this->getQuotedName($column)`, because column name can be a reserved keyword or contain some illegal characters.